docker/dockerfiles/be/be_prestop.sh

```diff
@@ -4,4 +4,4 @@ STARROCKS_ROOT=${STARROCKS_ROOT:-"/opt/starrocks"}
 STARROCKS_HOME=${STARROCKS_ROOT}/be
 
 # graceful stop be
-$STARROCKS_HOME/bin/stop_be.sh -g
+$STARROCKS_HOME/bin/stop_be.sh
```

Comment:
Why not call the `STARROCKS_HOME/bin/stop_be.sh` directly, instead of `be_prestop.sh`?

Reply:
it is a wrapper script to be used in the k8s operator for the pod prestop hook. need to make the command as simple and stable as possible.

Reply:
If the stop_be.sh is not stable, the prestop.sh seems also unstable?

Reply:
"stable" means the logic of the script. for example, if we want to have a graceful shutdown, we need to `stop_be.sh -g`, but we will make it in `pre_stop.sh` to call `stop_be.sh -g`, while to the operator, it still calls `prestop_be.sh` with no changes.